package.json

```diff
@@ -31,10 +34,25 @@
   },
   "homepage": "https://github.com/conveyal/lonlat",
   "devDependencies": {
-    "documentation": "^8.1.2",
-    "mastarm": "^4.4.0",
+    "@types/geojson": "^7946.0.8",
+    "@types/leaflet": "^1.7.3",
+    "@types/node": "^15.12.5",
+    "documentation": "^13.2.5",
+    "eslint": "^7.30.0",
     "semantic-release": "^15.10.3",
```

Comment:
Please update to the latest.

Reply:
Hasn't been updated...

Reply:
Should've wrote what happened here sorry -- the actual version in the yarn.lock is the newest, but I think since it's still compatible with the version here, the package.json hasn't changed. We could explicitly update the version here if you think that's a good idea?

Reply:
With `semantic-release` it's the same thing. The version is updated in the lock file, but `yarn upgrade` doesn't update the package.json because the latest version is still compatible. Should we explicitly update?

Reply:
Was doing the upgrades incorrectly, sorry. Should be fixed now!